packages/components/attach/src/Attach.stories.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Attach } from './Attach';
+
+storiesOf('ui/Attach', module)
```

Comment:
вот думаю, может компоненты из ДС вынести в отдельный список
```suggestion
storiesOf('Design System/Attach', module)
```

Reply:
Я ориентировался на https://github.com/gpn-prototypes/vega-ui/pull/50 Если переделывать, то во всех местах

Reply:
я бы не стал так выносить. Потому что как только мы чёт в обёртку добавим, даже самую мелочь эт уже не к дизайн системе будет относиться

Reply:
может быть